pipelines/managed/push-oot-kmods/push-oot-kmods.yaml

```diff
@@ -441,6 +441,10 @@ spec:
       runAfter:
         - sign-oot-kmods
     - name: push-oot-kmods
+      when:
+        - input: "$(params.buildRpm)"
```

Comment:
How is this supposed to work? The operator will not pass this parameter, so this will always rely on the default. So it's basically a constant and set to "yes". Do you expect to change the default in the future?